aryswebsite.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-20">
+    <link rel="stylesheet" href="main.css">
+    <title>Arianna Capizzi</title>
+  </head>
+  <body>
+    <h1 class="title">Arianna Capizzi</h1>
+    <p>Making it happen!</p>
+    <img src="https://scontent-lhr8-1.xx.fbcdn.net/v/t1.0-9/s960x960/75446652_10220512476583050_4849479348081655808_o.jpg?_nc_cat=102&_nc_ohc=ua-MGt4MOf0AX9oEp4x&_nc_ht=scontent-lhr8-1.xx&_nc_tp=7&oh=6567e1800e03e8d124f6a488d8595191&oe=5ECB1216" alt="Picture of me">
+  </body>
+</html>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<a href="https://www.facebook.com/arianna.izzicap" target="_blank">
+</a><a href="#" class="fa fa-facebook">
+    <div></div>
+</a>
+<a href="#" class="fa fa-twitter"></a>
+<a href="#" class="fa fa-linkedin"></a>
+<a href="#" class="fa fa-instagram"></a>

```

Comment:
It looks like the contents of this file has been moved to `index.html` and upgraded, if so can this file now be removed?